### PR TITLE
Export ideas in website

### PR DESCRIPTION
**Pull request by stijnvandervegt**
Add custom admin bar for Oauth admin users    
    - Add custom admin bar widget
    - Add button to export ideas
    - include admin bar in layout.html

I've include the widget of the custom admin bar in the layout.html. There must be a better way but I couldn't find it:
layout.html
`{% include '../lib/modules/custom-admin-widgets/views/widget.html' %}`

### CODE REVIEW

File: views/layout.html [hunk 1]
```diff
@@ -91,6 +91,9 @@
 
 {% endblock %}
 {% block beforeMain %}
+
+{% include '../lib/modules/custom-admin-widgets/views/widget.html' %}
+
```
Comment (rudivanhierden) [inline views/layout.html]: Isn't it better to create a helper function to include this HTML?

File: lib/modules/idea-overview-widgets/index.js [hunk 2]
```diff
@@ -1,23 +1,5 @@
...
-
-}
-
-const enrichIdeas = (ideas, googleMapsApiKey) => {
-  return ideas.map((idea) => {
-    idea.posterImageUrl = getPosterImageUrl(idea, googleMapsApiKey)
-    return idea;
-  });
-};
-
```
Comment (rudivanhierden) [inline lib/modules/idea-overview-widgets/index.js]: Why did you remove these lines?

Comment (stijnvandervegt) [reply lib/modules/idea-overview-widgets/index.js]: This is unused code, I already removed it once, but because of [this](https://github.com/Amsterdam/openstad-frontend/commit/cd2e6de29a531f7f746125231c977644a0513d99) merge it was back again :-)
